source/_integrations/calendar.markdown

```diff
@@ -180,3 +180,16 @@ data:
   start_date: "2022-10-01"
   end_date: "2022-10-02"
 ```
+
+Recent changes to Calendars in Home Assistant mean that zero duration Calendar events
```

Comment:
This is not a changelog, we should not write things like "Recent changes" for that reason.

Reply:
I've changed the text slightly.